bee-protocol/src/worker/propagator.rs

```diff
@@ -108,7 +103,7 @@ async fn propagate<B: StorageBackend>(
             }
 
             // Send child to the tip pool.
-            if let Err(e) = tip_tx.send((*message_id, parent1, parent2, ms_index_maybe)).await {
+            if let Err(e) = tip_tx.send((*message_id, parents, ms_index_maybe)).await {
```

Comment:
tbh, I think we should change this somewhat, because during syncing we don't want to send each just solidified messge to the tip-pool. I think we also want to do that only if we are withing BELOW_MAX_DEPTH as in the `index_updater`. This could improve syncing performance.

Reply:
I agree but this task also contains the solidification trigger atm so we would have to extract it. Do you want to do that ?

Reply:
We could just rename things a bit (`tip_tx`/`tip_rx` -> `solidified_tx`/`solidified_rx` etc) and limit the `tangle.insert_tip` call to happen only when with BELOW_MAX_DEPTH. Wdyt?

Reply:
I can do the change if you want and push to this PR.

Reply:
Sounds good to me. No side effects by not sending the tips, right ?

Reply:
There aren't any side effects when we don't add tips on synching. Testing currently the PR.